includes/apple-exporter/components/class-pre.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Apple_Exporter\Components;
+
+/**
+ * An HTML's blockquote representation.
+ *
+ * @since 0.2.0
+ */
+class Pre extends Component {
+
+  /**
+   * Look for node matches for this component.
+   *
+   * @param DomNode $node
+   * @return mixed
+   * @static
+   * @access public
+   */
+  public static function node_matches( $node ) {
+    if ( 'pre' == $node->nodeName ) {
+      return $node;
+    }
+
+    return null;
+  }
+
+  /**
+   * Build the component.
+   *
+   * @param string $text
+   * @access protected
+   */
+  protected function build( $text ) {
+    preg_match( '#<pre.*?>(.*?)</pre>#si', $text, $matches );
+    $text = $matches[1];
+
+    $this->json = array(
+      'role' => 'container',
+      'components' => array( array(
+        'role'   => 'body',
+        'text'   => $this->markdown->parse( $text ),
```

Comment:
I'd re-test this without parsing for markdown. I wonder if you actually want this if it's preformatted!